code/tests/test_app.py

```diff
@@ -688,7 +688,7 @@ def test_conversation_azure_byod_returns_500_when_exception_occurs(
         # then
         assert response.status_code == 500
         assert response.json == {
-            "error": "Exception in /api/conversation. See log for more details."
+            "error": "An error occurred. Please try again. If the problem persists, please contact the site administrator."
         }
 
```

Comment:
Can we add a unit test for the rate limit exception?

Reply:
Yes, we've added a unit test for the rate limit exception now.